migrations/02_cron-command_create_table_cron_migration.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace EE\Migration;
```

Comment:
Rename file as specified in - https://github.com/EasyEngine/auth-command/pull/9#discussion_r215918902